migrations/v6-MIGRATION.md

```diff
@@ -40,8 +40,6 @@ more thorough explanation of the new Google subscription model announced with Bi
 
 ### StoreProduct changes
 
-`StoreProduct` has been made an interface, which `GoogleStoreProduct` and `AmazonStoreProduct` implement.
```

Comment:
This is still true, right?

Reply:
Ha yup, did not mean to delete! Just added it back in 🙈